repositories.bzl

```diff
@@ -166,9 +159,69 @@ def rules_go():
         sha256 = "f04d2373bcaf8aa09bccb08a98a57e721306c8f6043a2a0ee610fd6853dcde3d",
     )
 
+
+def org_golang_x_tools():
```

Comment:
This and the following functions are already part of third_party_repositories.bzl